### DISCUSSION TITLE
Table with EditAction and several users

### DISCUSSION THREAD

**Discussion by photojo** [filamentphp/filament#10498] [Help]
What is the right way to avoid, that two users open, edit and save the same record at the same time?
(using filament 2)

**Comment by leandrocfe** [comment]
This? https://github.com/kenepa/resource-lock/tree/1.x

**Comment by photojo** [comment]
I tried to implement the plugin in my filament 2 app. But I think it only works with the admin panel? Right?

**Reply by ArtMin96** [reply]
[Livewire Real Time Presence](https://ahmedash.dev/blog/livewire/real-time-presence/) You can use this
